### DISCUSSION TITLE
Xaml 主页文件中的执行事件是否可以使用“切换登录方式”类型

### DISCUSSION THREAD

**Discussion by SuoHouDaiShi** [Meloong-Git/PCL#3034] [提问] [answered]
如题
<del>我这个整合包作者是真的不想让玩家点进版本设置去切换，搞不好出什么幺蛾子不得是我的事……</del>

**Comment by MoYuan-CN** [comment]
好像不行

**Comment by SuoHouDaiShi** [comment]
> ……要做切换登录方式的话 UI 估计不太好做，直接把那段 xaml 贴进去不知道行不行（应该不行
> 
> https://github.com/Hex-Dragon/PCL2/blob/main/Plain%20Craft%20Launcher%202/Pages/PageVersion/PageVersionSetup.xaml#L120-140

我的意思是指直接点一下按钮切换外置与正版，差不多这么写
```
"Type"="切换登录方式" "EventData"="仅正版登录|Littleskin登录"
```
然后换完之后直接右下角提示：当前登录方式为XXX

**Comment by WForst-Breeze** [comment, marked as answer]
意思是新添加一个 EventType 来支持切换登录方式吗，感觉在做完 #57 之后就没什么太大必要了……搜过了没有重复反馈，可以交一个看看

**Reply by SuoHouDaiShi** [reply]
提了 看看龙猫咋说
